localization/english/wc_draenor_l_english.yml

```diff
@@ -12,6 +12,6 @@
  RIFT_NAME:0 "Rift"
  
  activity_beyond_dark_portal:0 "Invasion of Draenor"
- activity_beyond_dark_portal_name:0 "[ACTIVITY.GetOwner.GetNamePossessiveOrMy|U] Invasion of Draenor"
+ activity_beyond_dark_portal_name:1 "[ACTIVITY.GetOwner.GetShortUINamePossessive|U] Invasion of Draenor"
  activity_beyond_dark_portal_desc:0 "Every brave men and women has been invited to end with the Dark Portal once and for all!"
```

Comment:
```suggestion
 activity_beyond_dark_portal_desc:0 "Every brave man and woman has been invited to end the Dark Portal once and for all!"
```

Reply:
That's nothing that was altered in this PR however...

Reply:
I'll transfer this suggestion to the localization branch. If you noticed a typo not related to the PR, create a [Issue](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/issues/new/choose).

Reply:
Done in 774798c65451a8b5de80033b5169907f1e245c3e.